Lob-API-public.yml

```diff
@@ -321,6 +321,106 @@ tags:
+      |-----------------------------------|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+      | `postcard.created`                | false     | A postcard is successfully created (Lob returns a 200 status code).                                                                                                                            |
+      | `postcard.rendered_pdf`           | false     | A postcard's PDF proof is successfully rendered.                                                                                                                                               |
+      | `postcard.rendered_thumbnails`    | false     | A postcard's thumbnails are successfully rendered.                                                                                                                                             |
+      | `postcard.deleted`                | false     | A postcard is successfully canceled.                                                                                                                                                           |
+      | `postcard.mailed`                 | true      | A postcard receives a "Mailed" [tracking event](https://docs.lob.com/#tracking_event_object). Only enabled for certain [Print & Mail Editions](https://dashboard.lob.com/#/settings/editions). |
+      | `postcard.in_transit`             | true      | A postcard receives an "In Transit" [tracking event](https://docs.lob.com/#tracking_event_object).                                                                                             |                                               
+      | `postcard.in_local_area`          | true      | A postcard receives an "In Local Area" [tracking event](https://docs.lob.com/#tracking_event_object).                                                                                          |
+      | `postcard.processed_for_delivery` | true      | A postcard receives a "Processed for Delivery" [tracking event](https://docs.lob.com/#tracking_event_object)                                                                                   |            
+      | `postcard.re-routed`              | true      | A postcard receives an "Re-Routed" [tracking event](https://docs.lob.com/#tracking_event_object).                                                                                              |
+      | `postcard.returned_to_sender`     | true      | A postcard receives a "Returned to Sender" [tracking event](https://docs.lob.com/#tracking_event_object)                                                                                       |                                                                                                                                  |
```

Comment:
Noting these need to be changed when tracking events get merged in. No point in changing them now and will make it easier to find later.

Reply:
yup, I'll change that once I get that PR merged